LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2023, The University of North Carolina at Chapel Hill
+All rights reserved.
```

Comment:
Should this be 2024? Noticed the audit one says 2016...

Reply:
A quick google search states it's supposed to match the year the work was first published.

Will update this and all other PRs.

Reply:
first commit was 2019

Reply:
done